kong/utils_test.go

```diff
@@ -567,6 +571,355 @@ func TestFillUpstreamsDefaults(T *testing.T) {
 	}
 }
 
+func getJSONSchemaFromFile(t *testing.T, filename string) Schema {
+	jsonFile, err := os.Open(filename)
+	if err != nil {
+		require.NoError(t, err)
+	}
+	defer jsonFile.Close()
+
+	var schema Schema
+	byteValue, err := io.ReadAll(jsonFile)
+	if err != nil {
+		require.NoError(t, err)
+	}
+	if err := json.Unmarshal(byteValue, &schema); err != nil {
+		require.NoError(t, err)
+	}
+	return schema
```

Comment:
To consider:

```suggestion
	jsonFile, err := os.Open(filename)
	require.NoError(t, err)
	defer jsonFile.Close()

	var schema Schema
	require.NoError(t, json.NewDecoder(jsonFile).Decode(&schema))
	return schema
```

An advantage of the above is that you don't need to read the whole file into memory at once.

Reply:
Nice, thanks. Fixed!